src/init.cpp

```diff
@@ -468,6 +468,7 @@ std::string HelpMessage(HelpMessageMode mode)
     if (showDebug) {
         strUsage += HelpMessageOpt("-acceptnonstdtxn", strprintf("Relay and mine \"non-standard\" transactions (%sdefault: %u)", "testnet/regtest only; ", !Params(CBaseChainParams::TESTNET).RequireStandard()));
         strUsage += HelpMessageOpt("-incrementalrelayfee=<amt>", strprintf(_("Fee rate (in %s/kB) used to define cost of free relay, used for mempool limiting and BIP 125 replacement. (default: %s)"), CURRENCY_UNIT, FormatMoney(DEFAULT_INCREMENTAL_RELAY_FEE)));
+        strUsage += HelpMessageOpt("-dustrelayfee=<amt>", strprintf(_("Fee rate (in %s/kB) used to defined dust, the size of an output such that it will cost about 1/3 of its value in fees at this fee rate to spend it. (default: %s)"), CURRENCY_UNIT, FormatMoney(DUST_RELAY_TX_FEE)));
```

Comment:
This should not be called dust*relay*fee, but dustfee. My reasoning is that dustfee has more implications than just setting a relay fee rate.